OpenRA.Mods.Common/Traits/Guard.cs

```diff
@@ -14,15 +14,14 @@
 using OpenRA.Graphics;
 using OpenRA.Mods.Common;
 using OpenRA.Mods.Common.Activities;
-using OpenRA.Mods.RA.Activities;
 using OpenRA.Traits;
 
-namespace OpenRA.Mods.RA.Traits
+namespace OpenRA.Mods.Common.Traits
 {
 	[Desc("The player can give this unit the order to follow and protect friendly units with the Guardable trait.")]
-	class GuardInfo : TraitInfo<Guard> { }
+	public class GuardInfo : TraitInfo<Guard> { }
 
-	class Guard : IResolveOrder, IOrderVoice
+	public class Guard : IResolveOrder, IOrderVoice
```

Comment:
Can we rename this to `Guards` ?
`Guard` sounds more like like an activity than a trait.